.github/workflows/pipeline.yaml

```diff
@@ -11,9 +11,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.9']
+        python-version: [ '3.6', '3.9' ]
     steps:
-      - uses: actions/checkout@v1
+      - name: "Install SSH key"
```

Comment:
What is this SSH key for?

Reply:
It has been mentioned in the group chat. You can search by this reference in the chat `https://github.com/rhos-infra/cibyl-e2e-zuul`. It's for the access.

Reply:
My bad, I missed that thread, thanks @adrianfusco